internal/service/apigateway/authorizer.go

```diff
@@ -187,7 +187,8 @@ func resourceAuthorizerRead(d *schema.ResourceData, meta interface{}) error {
 
 	d.Set("authorizer_credentials", authorizer.AuthorizerCredentials)
 
-	if authorizer.AuthorizerResultTtlInSeconds != nil { // nosem: helper-schema-ResourceData-Set-extraneous-nil-check
+	if authorizer.AuthorizerResultTtlInSeconds != nil { // nosemgrep:ci.helper-schema-ResourceData-Set-extraneous-nil-check
```

Comment:
This might be better off as either an enhancement to the semgrep rule to catch this pattern or to invert the test so as to avoid hitting the rule.